crates/nu-parser/src/parser.rs

```diff
@@ -4208,8 +4208,12 @@ pub fn parse_match_block_expression(working_set: &mut StateWorkingSet, span: Spa
             break;
         }
 
+        let connector = working_set
+            .get_span_contents(output[position].span)
+            .to_vec();
+        let mut connector = connector.as_slice();
```

Comment:
I'm not seeing why you needed to turn `connector` into a `vec` and then a slice here. Can you keep it as just a slice?

If it's an ownership thing, you can re-run `get_span_contents` a second time.

Reply:
found a way to remove `to_vec()`